lib/parse.rb

```diff
@@ -1,19 +1,31 @@
+#PARSE FILE
+#REQUIRING
+require 'json'
 #PARSE CLASS
 class Parse
     #INITIALIZE METHOD
    def initialize(path)
-     @path = path
+      @path = path
+      @lines = []
+      @line_count = 0
    end
-   #OPEN FILE METHOD
-   def open_file
-     if File.exists?(@path)
-       @content = File.open(@path, &:readline)
-     else
+   #COUNT LINES METHOD
+   def count_lines
+      if File.exists?(@path)
+        @content = File.open(@path, &:readlines)
+      else
       return nil
-     end
+      end
+      @line_count = @content.size
+      convert_json()
    end
-   #GET FIRST LINE METHOD
-   def get_first_line
-     @content
+   #CONVERT JSON METHOD
+   def convert_json
+      object = {
+        "#{@path}": {
+        lines: @line_count
```

Comment:
```suggestion
        lines: line_count()
```

Utilizando minha sugestão vc poderia chamar a função privada aqui